src/App.js

```diff
@@ -44,53 +44,65 @@ class App extends Component {
           </h1>
         </header>
         <main className="App-content">
-          <p>Введите свою заработную плату в месяц в теңге:</p>
-          <section className="App-form">
-            <input
-              className="App-input"
-              type="number"
-              min="52000"
-              step="any"
-              id="salary"
-              value={salary}
-              onChange={this.handleInputChange}
-            />
-            <label className="App-label" htmlFor="net">
-              На руки
-            </label>
-            <input
-              className="App-checkbox"
-              type="checkbox"
-              id="net"
-              checked={net}
-              onChange={this.handleCheckboxChange}
-            />
-          </section>
-          <hr />
-          <dl className="App-list">
-            <dt>Оклад:</dt>
-            <dd>{round(netSalary)}</dd>
-
-            <dt>
-              <abbr title="Обязательный пенсионный взнос">ОПВ</abbr>:
-            </dt>
-            <dd>{round(pension)}</dd>
-
-            <dt>
-              <abbr title="Индивидуальный подоходный налог">ИПН</abbr>:
-            </dt>
-            <dd>{round(tax)}</dd>
-
-            <dt>Заработная плата в месяц:</dt>
-            <dd>{round(grossSalary)}</dd>
-
-            <dt>Заработная плата за год:</dt>
-            <dd>{round(grossSalary * 12)}</dd>
-
-            <dt>Оклад за год:</dt>
-            <dd>{round(netSalary * 12)}</dd>
-          </dl>
-          <hr />
+          <section className="App-input-section">
+            <p>Введите свою заработную плату в месяц в теңге:</p>
+            <section className="App-form">
+              <input
+                className="App-input"
+                type="number"
+                min="52000"
+                step="any"
+                id="salary"
+                value={salary}
+                onChange={this.handleInputChange}
+                autoFocus
+              />
+              <label className="App-label" htmlFor="net">
+                На руки
+              </label>
+              <input
+                className="App-checkbox"
+                type="checkbox"
+                id="net"
+                checked={net}
+                onChange={this.handleCheckboxChange}
+              />
+            </section>
+          </section>          
+          <table className="App-list">
+            <tbody>  
+              <tr className="App-row">
+                <td>
+                  <p>Оклад</p>
+                  <p>{round(netSalary)}</p>
```

Comment:
I think this line could be separate table row.

Reply:
@jarjan Do you want Salary as seperate row? We aligned items as horizontally. Each table row has two items. (For example: Salary and Pension)

Reply:
@ramyamahi yes, separate each items to rows. This way we can remove `p` tag and put content insider `td` directly.

Reply:
Done